shuup/reporting/apps.py

```diff
@@ -0,0 +1,19 @@
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+import shuup.apps
+
+
+class AppConfig(shuup.apps.AppConfig):
+    name = "shuup.reporting"
+    provides = {
+        "admin_module": ["shuup.reporting.admin_module:ReportsAdminModule"],
+        "reports": [
+            "shuup.reporting.reports.sales:SalesReport",
+            "shuup.reporting.reports.sales:TotalSales",
+            "shuup.reporting.reports.sales:SalesPerHour",
```

Comment:
I think yes. Would make it easier to replace these initial reports with more advanced reporting.